sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java

```diff
@@ -19,36 +19,165 @@
+        DATE_OF_BIRTH_MONTH_COMBOBOX,
+        localDate.getMonth().getDisplayName(TextStyle.FULL, Locale.ENGLISH));
+    webDriverHelpers.selectFromCombobox(
+        DATE_OF_BIRTH_DAY_COMBOBOX, String.valueOf(localDate.getDayOfMonth()));
+  }
+
+  public void selectSex(String sex) {
+    webDriverHelpers.selectFromCombobox(SEX_COMBOBOX, sex);
+  }
+
+  public void fillNationalHealthId(String nationalHealthId) {
+    webDriverHelpers.fillInWebElement(NATIONAL_HEALTH_ID_INPUT, nationalHealthId);
+  }
+
+  public void fillPassportNumber(String passportNumber) {
+    webDriverHelpers.fillInWebElement(PASSPORT_NUMBER_INPUT, passportNumber);
+  }
+
+  public void fillPrimaryPhoneNumber(String primaryPhoneNumber) {
+    webDriverHelpers.fillInWebElement(PRIMARY_PHONE_NUMBER_INPUT, primaryPhoneNumber);
+  }
+
+  public void fillPrimaryEmailAddress(String primaryPhoneNumber) {
+    webDriverHelpers.fillInWebElement(PRIMARY_EMAIL_ADDRESS_INPUT, primaryPhoneNumber);
+  }
+
+  public void selectReturningTraveler(String option) {
+    String optionBox = "//div[@id='returningTraveler']//label[contains(text(),'" + option + "')]";
+    By optionBoxXpath = By.xpath(optionBox);
+    webDriverHelpers.clickOnWebElementBySelector(optionBoxXpath);
+  }
+
+  public void fillDateOfReport(LocalDate date) {
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("M/dd/yyyy");
+    webDriverHelpers.clearWebElement(DATE_OF_REPORT_INPUT);
+    webDriverHelpers.fillInWebElement(DATE_OF_REPORT_INPUT, formatter.format(date));
+  }
+
+  public void fillDiseaseOfSourceCase(String diseaseOrCase) {
+    webDriverHelpers.selectFromCombobox(DISEASE_OF_SOURCE_CASE_COMBOBOX, diseaseOrCase);
+  }
+
+  public void fillCaseIdInExternalSystem(String externalId) {
+    webDriverHelpers.fillInWebElement(CASE_ID_IN_EXTERNAL_SYSTEM_INPUT, externalId);
+  }
+
+  public void fillDateOfLastContact(LocalDate date) {
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("M/dd/yyyy");
+    webDriverHelpers.fillInWebElement(DATE_OF_LAST_CONTACT_INPUT, formatter.format(date));
+  }
+
+  public void fillCaseOrEventInformation(String caseOrEventInfo) {
+    webDriverHelpers.fillInWebElement(CASE_OR_EVENT_INFORMATION_INPUT, caseOrEventInfo);
+  }
+
+  public void selectResponsibleRegion(String selectResponsibleRegion) {
+    webDriverHelpers.selectFromCombobox(RESPONSIBLE_REGION_COMBOBOX, selectResponsibleRegion);
+  }
+
+  public void selectResponsibleDistrict(String responsibleDistrict) {
+    webDriverHelpers.selectFromCombobox(RESPONSIBLE_DISTRICT_COMBOBOX, responsibleDistrict);
+  }
+
+  public void selectResponsibleCommunity(String responsibleCommunity) {
+    webDriverHelpers.selectFromCombobox(RESPONSIBLE_COMMUNITY_COMBOBOX, responsibleCommunity);
+  }
+
+  public void selectTypeOfContact(String type) {
+    String typeOfContact = "//label[contains(text(), '" + contact.getTypeOfContact() + "')]";
+    By typeOfContactXpath = By.xpath(typeOfContact.toString());
```

Comment:
Please identify specific css selector for labels and use the `clickWebElementByText` method

Reply:
done